packages/esbuild-plugin/src/index.js

```diff
@@ -0,0 +1,118 @@
+/**
```

Comment:
We're starting to add types to the packager plugins. See if you're up to the task of adding Flow types. If not, I'll do it after the fact.